.github/workflows/publish_docker_images_cron.yml

```diff
@@ -2,7 +2,7 @@ name: Schedule Build Docker Images For Main Images And Nightly
 
 on:
   schedule:
-    - cron: '0 0 * * *'
+    - cron: 30 0 * * *
```

Comment:
I understand how we are conditioned to schedule cron jobs around midnight, even though this setting is in UTC time.

( FYI, I am writing this in 00:30 UTC = 3:30 TRT )

Reply:
By the way cron schedules trigger only main branch. They are not triggered on non-main branches https://stackoverflow.com/questions/63436541/github-action-workflow-schedule-not-working-on-non-default-branch